chef-kitchen/apt/debian8/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:8
+LABEL maintainer="Agent Platform <team-agent-platform@datadoghq.com>"
+
+# The Debian Jessie GPG key expired on Sat Nov 19 2022 21:01:13 GMT+0000.
+# Use gpgv wrapper that ignores key expiration date but checks package signatures.
+COPY gpgvnoexpkeysig /usr/local/sbin
+RUN echo 'Dir::Bin::gpg "/usr/local/sbin/gpgvnoexpkeysig";' >> /etc/apt/apt.conf.d/20datadog
+
+CMD [ "/root/start.sh" ]
```

Comment:
This is a copy-paste error from the CentOS images that should be removed: there is no `/root/start.sh` script in this image.

Reply:
Right, my bad ! See #90